core/slice.go

```diff
@@ -15,10 +15,10 @@ import (
 	"github.com/dominant-strategies/go-quai/core/rawdb"
 	"github.com/dominant-strategies/go-quai/core/types"
 	"github.com/dominant-strategies/go-quai/core/vm"
-	"github.com/dominant-strategies/go-quai/ethclient/quaiclient"
 	"github.com/dominant-strategies/go-quai/ethdb"
 	"github.com/dominant-strategies/go-quai/log"
 	"github.com/dominant-strategies/go-quai/params"
+	"github.com/dominant-strategies/go-quai/quaiclient"
```

Comment:
is this a vestige of another PR? I don't mind us tidying up modules imports as needed, but its surprising that this was needed in this PR. Want to make sure this isn't a sign of a rebase which went awry which might have other issues too